### ISSUE TITLE
E_STRICT error caused by using non-variable as reference

### ISSUE THREAD

**Issue by elsand** [bambora/checkout-woocommerce#6]
```php
$item_tax_rate = array_shift($wc_tax->get_rates($item_tax_class));
```
on line 325 in woocommerce-gateway-bambora.php causes an E_STRICT error, as only variables can be passed by reference.

**Comment by AllanWLie** [comment]
Thank you for reporting this issue. We will look into it and consider it for the next release. In the meantime feel free to make a pull request and implement the issues.

**Comment by sorenla** [comment]
Fixed in version 1.2.8
